source/wp-content/themes/wporg-developer-2023/src/dashicons-page/render.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace WordPressdotorg\Theme\Developer_2023\Block_Dashicons_Page;
+
+require_once dirname( __DIR__, 2 ) . '/inc/dashicons.php';
+
+$icons          = array();
+$icons_sections = array();
+
+foreach ( \DevHub_Dashicons::get_dashicons() as $section_group => $section ) {
+	$icon_section = array(
+		'label' => $section['label'],
+		'slug'  => sanitize_title( $section_group ),
+		'icons' => array(),
+	);
+	foreach ( $section['icons'] as $k => $v ) {
+		$icons[ $k ]                 = $v;
+		$icons[ $k ]['sectionLabel'] = $section['label'];
+		$icon_section['icons'][]     = $k;
+	}
+	$icons_sections[] = $icon_section;
+}
+
+$selected_icon = $_GET['icon'] ?? '';
```

Comment:
This line flags some security lint issues, but it's OK since the value is matched against the icon list. I'd add a comment disabling the check to avoid those false errors.

```suggestion
$selected_icon = $_GET['icon'] ?? ''; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- Value checked against icon safelist.
```